plugins/package-managers/bazel/src/test/kotlin/BazelTest.kt

```diff
@@ -40,4 +47,19 @@ class BazelTest : WordSpec({
             result shouldBe "7.0.1"
         }
     }
+
+    "Bazel package manager" should {
+        "support local registry" {
```

Comment:
This test only covers selecting the correct implementation of `BazelModuleRegistryService` given a registry URL from the lock file. Is this intended?

In order to also test retrieving package metadata, the synthetic `MODULE.bazel` file would need to declare a dependency for which metadata is present in the local registry.

I'm not sure it's worth it given how simple the implementation of `LocalBazelModuleRegistryService.getModuleMetadata()` and `LocalBazelModuleRegistryService.getModuleSourceInfo()` is.

Reply:
Yes this was the intention of the test: to show that local registries are not breaking ORT (as they did before, throwing an OkHttp error).

Extra tests regarding the package metadata should be written in the future.